mitmproxy/addons/termlog.py

```diff
@@ -5,14 +5,19 @@
 
 
 class TermLog:
-    def __init__(self, outfile=sys.stdout):
+    def __init__(self, outfile=None):
         self.options = None
         self.outfile = outfile
 
     def configure(self, options, updated):
         self.options = options
 
     def log(self, e):
+        if self.outfile is None:
+            if log.log_tier(e.level) == log.log_tier("error"):
+                self.outfile = sys.stderr
+            else:
+                self.outfile = sys.stdout
```

Comment:
The purpose of this PR is that all log messages marked as `error` are written to stderr.
However, if the very first message after program start is e.g. `info` you set `self.outfile = sys.stdout`.

This means the whole code block is only executed once, and after that either stdout or stderr is fixed for all following messages. IMHO this is not intended. Every new message should be evaluated again.